sql/timeseries/cruxFastInp.sql

```diff
@@ -0,0 +1,31 @@
+#standardSQL
+# Small CLS by device
+
+CREATE TEMP FUNCTION IS_GOOD (good FLOAT64, needs_improvement FLOAT64, poor FLOAT64) RETURNS BOOL AS (
+  good / (good + needs_improvement + poor) >= 0.75
+);
+
+CREATE TEMP FUNCTION IS_NON_ZERO (good FLOAT64, needs_improvement FLOAT64, poor FLOAT64) RETURNS BOOL AS (
+  good + needs_improvement + poor > 0
+);
+
+SELECT
+  REGEXP_REPLACE(CAST(yyyymm AS STRING), '(\\d{4})(\\d{2})', '\\1_\\2_01') AS date,
+  UNIX_DATE(CAST(REGEXP_REPLACE(CAST(yyyymm AS STRING), '(\\d{4})(\\d{2})', '\\1-\\2-01') AS DATE)) * 1000 * 60 * 60 * 24 AS timestamp,
+  IF(device = 'desktop', 'desktop', 'mobile') AS client,
+  SAFE_DIVIDE(
+    COUNT(DISTINCT IF(IS_GOOD(fast_inp, avg_inp, slow_inp), origin, NULL)),
+    COUNT(DISTINCT IF(IS_NON_ZERO(fast_inp, avg_inp, slow_inp), origin, NULL))
+  ) * 100 AS percent
+FROM
+  `chrome-ux-report.materialized.device_summary`
+WHERE
+  device IN ('desktop', 'phone') AND
+  yyyymm >= 202204
```

Comment:
Data exists for the previous `responsiveness` metric under this field name back to 202202. Slightly different implementation but still good to track historical progress.
```suggestion
  yyyymm >= 202202
```

Reply:
Might also give us something more interesting to graph!